localization/english/events/event_localization/spark_council_events_l_english.yml

```diff
@@ -3,17 +3,17 @@
   # New Magi is recruited
   aeluran_advisor_task.0101.t:0 "A New Magi Joins Our Ranks!"
   aeluran_advisor_task.0101.desc:0 "A powerful Magi known as %NEW_MAGI% has been convinced by [councillor.GetTitledFirstName] to join your Magi Core!"
-	aeluran_advisor_task.0101.a:0 "Well done [councillor.GetTitledFirstName]! We grow ever more powerful"
-	aeluran_advisor_task.0101.b:0 "We dont need such a Magi, send them away"
+	aeluran_advisor_task.0101.a:0 "Well done [councillor.GetTitledFirstName]! We grow ever more powerful."
+	aeluran_advisor_task.0101.b:0 "We dont need such a Magi, send them away."
 
   # advisor heals child of devotee_target
   aeluran_advisor_task.01021.t:0 "Religious Exaltation!"
   aeluran_advisor_task.01021.desc:0 "Its a miracle! The child of [devotee_target.GetName] has been healed by [councillor.GetTitledFirstName]!"
-	aeluran_advisor_task.01021.a:0 "Well done [councillor.GetTitledFirstName]! They will owe a lifetime of gratitude"
+	aeluran_advisor_task.01021.a:0 "Well done [councillor.GetTitledFirstName]! They owe us a lifetime of gratitude."
```

Comment:
maybe just drop the "us", its really the Aeluran Advisor they owe the gratitude even if its you that gets the hook.

Or maybe "Their gratitude will last a lifetime"